homeassistant/components/v2c/strings.json

```diff
@@ -47,6 +50,49 @@
       },
       "fv_power": {
         "name": "Photovoltaic power"
+      },
+      "battery_power": {
+        "name": "Battery power"
+      },
+      "slave_error": {
+        "name": "Slave error",
+        "state": {
+          "no_error": "No error",
+          "communication": "Communication",
+          "reading": "Reading",
+          "slave": "Slave",
+          "waiting_wifi": "Waiting WiFi",
+          "waiting_communication": "Waiting communication",
+          "wrong_ip": "Wrong IP",
+          "slave_not_found": "Slave not found",
+          "wrong_slave": "Wrong slave",
+          "no_response": "No response",
+          "clamp_not_connected": "Clamp not connected",
+          "illegal_function": "Illegal function",
+          "illegal_data_address": "Illegal data address",
+          "illegal_data_value": "Illegal data value",
+          "server_device_failure": "Server device failure",
+          "acknowledge": "Acknowledge",
+          "server_device_busy": "Server device busy",
+          "negative_acknowledge": "Negative acknowledge",
+          "memory_parity_error": "Memory parity error",
+          "gateway_path_unavailable": "Gateway path unavailable",
+          "gateway_target_no_resp": "Gateway target no response",
+          "server_rtu_inactive244_timeout": "Server RTU inactive/timeout",
+          "invalid_server": "Invalid server",
+          "crc_error": "CRC error",
+          "fc_missmatch": "FC missmatch",
+          "server_id_missmatch": "Server id missmatch",
+          "packet_length_error": "Packet length error",
+          "parameter_count_error": "Parameter count error",
+          "parameter_limit_error": "Parameter limit error",
+          "request_queue_full": "Request queue full",
+          "illegal_ip_or_port": "Illegal IP or port",
+          "ip_connection_failed": "IP connection failed",
+          "tcp_head_missmatch": "TCP head missmatch",
```

Comment:
```suggestion
          "server_id_missmatch": "Server id mismatch",
          "packet_length_error": "Packet length error",
          "parameter_count_error": "Parameter count error",
          "parameter_limit_error": "Parameter limit error",
          "request_queue_full": "Request queue full",
          "illegal_ip_or_port": "Illegal IP or port",
          "ip_connection_failed": "IP connection failed",
          "tcp_head_missmatch": "TCP head mismatch",
```

Reply:
Maybe these should be fixed in the lib instead

Reply:
Version bumped https://github.com/dgomes/pytrydan/compare/v0.6.0...v0.6.1